unicodetools/org/unicode/tools/emoji/CandidateData.java

```diff
@@ -491,15 +488,16 @@ private void addCombosWithGenderAndSkin(String source) {
 		|| EmojiData.EMOJI_DATA_BETA.getModifierBasesRgi().containsSome(source);
 	UnicodeSet all_Emoji_Modifier_Base = null;
 	String fromNames = names.get(source);
-	if (hasModifierBase) {
-	    // find the point where it occurs; not efficient but we don't care
-	    all_Emoji_Modifier_Base = new UnicodeSet(emoji_Modifier_Base)
-		    .addAll(EmojiData.EMOJI_DATA_BETA.getModifierBases())
-		    .remove("🤝") // special hack to remove skin color
-		    .freeze();
-
-	    addCombos(source, fromNames, "", source, "", ": ", all_Emoji_Modifier_Base, "", "");
-	}
+	// disable this, so that all gender variants come from the file instead of being manufactured.
```

Comment:
I used to "flesh out" the candidates by adding the skintones. That turns out to be more an more complicated, so backing that out and supplying the list explicitly.